[camel k] cannot find atlas component

## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Documentation issue or request
</code></pre>

## Description

Get this when running a integration with mapping.

```
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz Caused by: org.apache.camel.ResolveEndpointFailedException: Failed to resolve endpoint: atlas://mapping-flow-0-step-1.json?encoding=UTF-8&sourceMapName=Syndesis.CAPTURED_OUT_MESSAGES_MAP due to: No component found with scheme: atlas
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.impl.DefaultCamelContext.getEndpoint(DefaultCamelContext.java:759)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.util.CamelContextHelper.getMandatoryEndpoint(CamelContextHelper.java:80)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.RouteDefinition.resolveEndpoint(RouteDefinition.java:227)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.impl.DefaultRouteContext.resolveEndpoint(DefaultRouteContext.java:116)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.impl.DefaultRouteContext.resolveEndpoint(DefaultRouteContext.java:122)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.SendDefinition.resolveEndpoint(SendDefinition.java:62)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.SendDefinition.createProcessor(SendDefinition.java:56)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.ProcessorDefinition.createProcessor(ProcessorDefinition.java:518)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.ProcessorDefinition.createOutputsProcessorImpl(ProcessorDefinition.java:481)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.ProcessorDefinition.createOutputsProcessor(ProcessorDefinition.java:448)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.ProcessorDefinition.createOutputsProcessor(ProcessorDefinition.java:186)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.ProcessorDefinition.createChildProcessor(ProcessorDefinition.java:205)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.PipelineDefinition.createProcessor(PipelineDefinition.java:51)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.ProcessorDefinition.makeProcessorImpl(ProcessorDefinition.java:569)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.ProcessorDefinition.makeProcessor(ProcessorDefinition.java:530)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.ProcessorDefinition.addRoutes(ProcessorDefinition.java:240)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  at org.apache.camel.model.RouteDefinition.addRoutes(RouteDefinition.java:1349)
i-l-idqznfgqs09i-ddlgz-6cccbb6c46-swqqb i-l-idqznfgqs09i-ddlgz  ... 15 more
```

--- a/app/integration/runtime-camelk/pom.xml
+++ b/app/integration/runtime-camelk/pom.xml
@@ -63,6 +63,13 @@
       <artifactId>camel-core</artifactId>
     </dependency>
 
+    <!-- set atlasmap as always required runtime dependency-->
+    <dependency>
+      <groupId>io.atlasmap</groupId>
+      <artifactId>camel-atlasmap</artifactId>
+      <scope>runtime</scope>
+    </dependency>
+
     <!-- camel-k -->
     <dependency>
       <groupId>org.apache.camel.k</groupId>
@@ -79,11 +86,6 @@
     <dependency>
       <groupId>org.apache.camel</groupId>
       <artifactId>camel-test</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>io.atlasmap</groupId>
-      <artifactId>camel-atlasmap</artifactId>
       <scope>test</scope>
     </dependency>
     <dependency>

